docs/content/preview/explore/change-data-capture/cdcsdk-server.md

```diff
@@ -0,0 +1,154 @@
+
+The archive has the following layout:
+
+```output
+cdcsdk-server
+  |--CHANGELOG.md
+  |-- conf
+  |-- CONTRIBUTE.md
+  |-- COPYRIGHT.txt
+  |-- debezium-server-1.9.2.Final-runner.jar
+  |-- lib
+  |-- LICENSE-3rd-PARTIES.txt
+  |-- LICENSE.txt
+  |-- README.md
+  |-- run.sh
+```
+
+#### Unpack and run instructions
+
+```sh
+export CDCSDK_VERSION=<x.y.z>
+wget https://github.com/yugabyte/cdcsdk-server/releases/download/v${CDCSDK_VERSION}/cdcsdk-server-dist-${CDCSDK_VERSION}.tar.gz
+
+# Or if you are using GitHub CLI
+gh release download v{CDCSDK_VERSION} -A tar.gz --repo yugabyte/cdcsdk-server
+
+tar xvf cdcsdk-server-dist-${CDCSDK_VERSION}.tar.gz
+cd cdcsdk-server
+
+# Configure the application. More details in the next section
+touch conf/application.properties
+
+# Run the application
+./run.sh
+```
+
+### Configuration
+
+The main configuration file is `conf/application.properties`. There are multiple sections configured:
+* `debezium.source`: is for source connector configuration. Each instance of Debezium Server runs exactly one connector.
+* `debezium.sink` is for the sink system configuration.
+* `debezium.format` is for the output serialization format configuration.
+* `debezium.transforms` is for the configuration of message transformations.
+
+**Example:**
+
+```properties
+debezium.sink.type=kafka
+debezium.sink.kafka.producer.bootstrap.servers=127.0.0.1:9092
+debezium.sink.kafka.producer.key.serializer=org.apache.kafka.common.serialization.StringSerializer
+debezium.sink.kafka.producer.value.serializer=org.apache.kafka.common.serialization.StringSerializer
+debezium.source.connector.class=io.debezium.connector.yugabytedb.YugabyteDBConnector
+debezium.source.database.hostname=127.0.0.1
+debezium.source.database.port=5433
+debezium.source.database.user=yugabyte
+debezium.source.database.password=yugabyte
+debezium.source.database.dbname=yugabyte
+debezium.source.database.server.name=dbserver1
+debezium.source.database.streamid=de362081fa864e94b35fcac6005e7cd9
+debezium.source.table.include.list=public.test
+debezium.source.database.master.addresses=127.0.0.1:7100
+debezium.source.snapshot.mode=never
+```
+
+The `debezium.source` configurations are nothing but the Debezium Connector's configurations only where you can specify the `debezium.source` as a prefix to any of the connector's configurations. For a complete list of the Debezium configurations, see [Debezium Connector for YugabyteDB configurations](../change-data-capture/debezium-connector-yugabytedb.md#connector-configuration-properties).
+
+<!-- TODO Vaibhav: add more configuration examples -->
+
+#### Configuration using HTTP client
+
+Configuration using environment variables maybe useful when running in containers. The rule of thumb is to convert the keys to UPPER CASE and replace `.` with `_`. For example, `debezium.source.database.port` has to be changed to `DEBEZIUM_SOURCE_DATABASE_PORT`.
```

Comment:
This line should be placed in the previous section. Remove the heading `Configuration using HTTP client`

Reply:
Done.